tsconfig.release.json

```diff
@@ -1,9 +1,8 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "module": "es2020",
-    "target": "es2020",
-    "lib": ["ES2020", "ESNext"],
+    "module": "ESNext",
+    "target": "ES2021",
```

Comment:
In `scripts/build.ts`, you use `ES2022` L214 and on L240, but here you use `ES2021`. Don’t you want to use the same target? :thinking:

Reply:
Fixed 👍